torch/testing/_internal/common_optimizers.py

```diff
@@ -508,7 +507,7 @@ def optim_inputs_func_lbfgs(device=None):
         OptimizerInput(params=None, kwargs={}, desc="default"),
         OptimizerInput(params=None, kwargs={"lr": 0.01}, desc="non-default lr"),
         OptimizerInput(
-            params=None, kwargs={"tolerance_grad": math.inf}, desc="tolerance_grad"
+            params=None, kwargs={"tolerance_grad": 1e-6}, desc="tolerance_grad"
```

Comment:
If it's inf, lbfgs will think it should terminate as it's the most optimal already.